pp/integrations/plugin-packs/procedures/applications-pacemaker-ssh.md

```diff
@@ -2,40 +2,431 @@
 id: applications-pacemaker-ssh
 title: Pacemaker
 ---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## Pack assets
+
+### Templates
+
+The Monitoring Connector **Pacemaker** brings a host template:
+
+* **App-Pacemaker-SSH-custom**
+
+The connector brings the following service templates (sorted by the host template they are attached to):
+
+<Tabs groupId="sync">
+<TabItem value="App-Pacemaker-SSH-custom" label="App-Pacemaker-SSH-custom">
+
+| Service Alias | Service Template                     | Service Description                                             |
+|:--------------|:-------------------------------------|:----------------------------------------------------------------|
+| CRM           | App-Pacemaker-CRM-SSH-custom         | Check cluster state with crm_mon command                        |
+| Clustat       | App-Pacemaker-Clustat-SSH-custom     | Check cluster state with clustat command                        |
+| Constraints   | App-Pacemaker-Constraints-SSH-custom | Check if there are any constraints applied to a cluster resource |
+
+> The services listed above are created automatically when the **App-Pacemaker-SSH-custom** host template is used.
+
+</TabItem>
+</Tabs>
+
+### Collected metrics & status
+
+Here is the list of services for this connector, detailing all metrics linked to each service.
+
+<Tabs groupId="sync">
+<TabItem value="CRM" label="CRM">
+
+| Metric name                                             | Unit  |
+|:--------------------------------------------------------|:------|
+| *cluster*~connection-status                             | N/A   |
+| *cluster*~quorum-status                                 | N/A   |
+| *cluster*~cluster.nodes.online.count                    | count |
+| *cluster*~cluster.nodes.offline.count                   | count |
+| *cluster*~cluster.nodes.standby.count                   | count |
+| *cluster*~cluster.actions.failed.count                  | count |
+| *resources*#resource-status                             | N/A   |
+| *resources*#resource.actions.failed.count               | count |
+| *resources*#resource.migration.failed.count             | count |
+| *clone_resources*#clone-resource-status                 | N/A   |
+| *clone_resources*#clone_resource.actions.failed.count   | count |
+| *clone_resources*#clone_resource.migration.failed.count | count |
+
+</TabItem>
+<TabItem value="Clustat" label="Clustat">
+
+| Metric name    | Unit  |
```

Comment:
```suggestion
> This service template is deprecated and is no longer provided with the latest versions of the connector since the `clustat` is no longer part of Pacemaker's commande-line tools since RHEL7.

| Metric name    | Unit  |
```